src/main/resources/application.properties

```diff
@@ -0,0 +1,4 @@
+spring.datasource.url=jdbc:neo4j:bolt://localhost
+spring.datasource.driver-class-name=org.neo4j.jdbc.bolt.BoltDriver
+spring.datasource.username=neo4j
+spring.datasource.password=toto
```

Comment:
for another review: it would be nice if we could prompt the user/password when the `main` method runs, but that's out of scope, of course :-)